target/linux/lantiq/xrx200_legacy/base-files/etc/board.d/02_network

```diff
@@ -57,7 +57,7 @@ lantiq_setup_macs()
 		;;
 	tplink,tdw8970|\
 	tplink,tdw8980)
-		wan_mac=$(macaddr_add "$(mtd_get_mac_binary boardconfig 0xf100)" 1)
+		wan_mac=$(macaddr_add "$(mtd_get_mac_binary boardconfig 0x100)" 1)
```

Comment:
This can be handled in dts, no?

Reply:
Oh right. This is for an interface not specified in dts. nvm.

Reply:
yeah, it's for the dsl interface.

Reply:
Looking at this further, there’s the vdsl node. If the driver uses of_get_mac_address, it can be converted.

Reply:
I don't know. I checked the other lantiq based devices, but I couldn't find any device that sets the mac address of the dsl port through dts.

Reply:
i checked the driver. Not happening.